changelogs/fragments/263-new_mergelist_plugin.yml

```diff
@@ -0,0 +1,2 @@
+features:
```

Comment:
1. This is not a valid section.
2. New plugins and modules must not have a changelog fragment.

Reply:
Thanks! Removed